ui/components/multichain/pages/send/components/your-accounts.tsx

```diff
@@ -25,7 +27,12 @@ export const SendPageYourAccounts = () => {
   // Your Accounts
   const accounts = useSelector(getUpdatedAndSortedAccounts);
   const internalAccounts = useSelector(getInternalAccounts);
-  const mergedAccounts = mergeAccounts(accounts, internalAccounts);
+  const mergedAccounts: InternalAccount &
+    { keyring: KeyringTypes; label: string }[] = useMemo(() => {
+    return mergeAccounts(accounts, internalAccounts).filter(
+      (account: InternalAccount) => account.type !== BtcAccountType.P2wpkh,
```

Comment:
What about re-using the same pattern here with `allowedAccountTypes`? and the default would be EOA + 4337 account types.

Reply:
Sure, let me change that.